### PR TITLE
Added a test case

### PR DESCRIPTION
**Pull request by arturb90**

### CODE REVIEW

File: javaparser-testing/src/test/java/com/github/javaparser/SetupTest.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+import org.junit.Test;
+
+public class SetupTest {
+
+	@Test
+	public void test() {
+		
+	}
+
```
Comment (matozoid) [inline javaparser-testing/src/test/java/com/github/javaparser/SetupTest.java:34]: Best PR ever :-D

**Comment by ftomassetti** [comment]
Should we just close it? @arturb90 I suspect you submitted this one by mistake but if you need help we are here, just ask

**Comment by matozoid** [comment]
Yeah, let's :-)

**Comment by arturb90** [comment]
I did the Eclipse Setup Guide. Sorry if i got it wrong!

**Comment by matozoid** [comment]
No problem, @ryan-beckett - maybe this is useful feedback for the guide?

**Comment by ryan-beckett** [comment]
@matozoid, @arturb90: The guide states clearly not to create a pull request unless you have a real contribution. However, I'm glad it worked for you arturb!

**Pull request closed without merging**